app/presenters/documents_info_export_presenter.rb

```diff
@@ -0,0 +1,86 @@
+class DocumentsInfoExportPresenter
```

Comment:
Reflecting on this comment, my suggestion that this acts like a model is incorrect as I realise it's actually an array of document_ids rather than a single one. So rather than acting like a model that's looked up it's more a class that is used to perform a transformation.

Reply:
As per our conversation f2f, we decided the best place was to put this is in the whitehall/exporters directory.